routes/api/accounts.js

```diff
@@ -0,0 +1,53 @@
+const express = require( 'express' )
+
+const bcrypt = require( 'bcrypt' )
+const encryptPassword = require( '../../auth/encryptPassword' )
+const RegistrationEmail = require( '../../src/mail/registration_email.js' )
+const jwt = require( 'jsonwebtoken' )
+const { testForCode, whereClause } = require( '../accounts/verify_user' )
+
+const router = express.Router()
+
+const ERROR_MESSAGE = "Incorrect email or password. Have you verified your email?"
+const ERROR_MESSAGEB = "Email not valid"
+
+router.post( '/register', ( request, response, next ) => {
```

Comment:
Much of the behavior in this file seems to be very similar to functionality already present in other places in the codebase.  Is there a way we can share this implementation to avoid code duplication?